app/recordtransfer/jobs.py

```diff
@@ -61,18 +61,23 @@ def create_downloadable_bag(submission: Submission, user_triggered: User) -> Non
                 new_job.save()
                 return
 
-        zipf = None
+        temp_zip_path = None
```

Comment:
I'm going to suggest we leave the file open when calling `mkstemp`. This will require having to have a handle to the file in the `finally:` block.

```suggestion
        temp_zip_path = None
        temp_zip_fd = None
```